source/blog/2016-05-30-dropping-dropbox.html.markdown

```diff
@@ -0,0 +1,97 @@
+---
+title: Dropping Dropbox
+date: 2016-05-30 16:23:22 UTC
+og_image: /img/blog/dropping-dropbox/dashboard.png
+background_image: /img/blog/dropping-dropbox/background.jpg
+background_image_source: https://twitter.com/CloudyConway/status/737279203532541952
+---
+
+Dropbox is a jewel of the Y-Combinator industrial complex: a successful company that provides software as a service to ordinary people. They even allegedly [turned down an acquisition offer from Steve Jobs](http://techcrunch.com/2011/10/18/dropbox-said-no-to-nine-digits-acquisition-offer-from-apple-steve-jobs/). Their success is no small feat, but sadly it appears that they had to make a deal with the devil to achieve so much. 
+
+From my perspective, the company has been acting suspiciously for a while. Appointing [George W. Bush's Secretary of State to their Board](http://bits.blogs.nytimes.com/2014/04/18/protests-continue-against-dropbox-after-appointing-condoleezza-rice-to-board/) was a big red flag. It inspired a whole [Drop Dropbox](http://www.drop-dropbox.com/) movement. I've been uneasy about Dropbox, but when they announced they'd be integrating in with my [operating system's kernel](https://blogs.dropbox.com/tech/2016/05/going-deeper-with-project-infinite/), I decided to move away from them. 
+
+<!-- more -->
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Things I don&#39;t want: <a href="https://twitter.com/hashtag/Dropbox?src=hash">#Dropbox</a> up in my kernel. <a href="https://t.co/p4LV87cKYy">pic.twitter.com/p4LV87cKYy</a></p>&mdash; Dusty (@duspom) <a href="https://twitter.com/duspom/status/735263193497505792">May 25, 2016</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+I've been trying to move away from online services that allow easy mass surveillance. For example, I started in 2012, when I [moved off of Gmail](https://ashfurrow.com/blog/switching-from-gmail-to-fastmail/). It's not been easy, I think in part because services that are popular are typically ad-supported – viewing your data is how they sell ads to you – this implies a lack of end-to-end encryption, and security in general. I've been tolerating Dropbox because it's _so damm convenient_, but the kernel extension was the last straw, so I [asked for alternatives](https://twitter.com/ashfurrow/status/736936037876895744). 
+
+The solution that appealed to me the most was [BitTorrent Sync](https://getsync.com). It's closed source (boo) but offers a good compromise between security and convenience. I decided to give it a try, and in the process I ran into confusing documentation and advice that would make me _less_ secure than using Dropbox. It really sucks that in order to _securely_ do something as commonplace as syncing files, you have to be pretty tech savvy. 
```

Comment:
Good question. I didn't, but only because of problems I've had using iCloud with documents before. It's possible the usability and stability problems have been fixed.